app/views/foreman_salt/job_templates/salt_run_apply_highstate_-_ssh_default.erb

```diff
@@ -0,0 +1,19 @@
+<%#
+name: Salt Run apply.highstate - SSH default
+job_category: Salt
+description_format: Run Salt apply.highstate on host
+feature: foreman_salt_run_apply_highstate
+provider_type: SSH
+snippet: false
+template_inputs:
+- name: test-run
+  required: true
+  input_type: user
+  description: Enables test run
+  options: "no\r\nyes"
+  advanced: true
+kind: job_template
+%>
+
+salt-call state.highstate \
+<%= ' test=true' if input('test-run') == 'yes' -%>
```

Comment:
This template is a specialization of the `Salt Run Function` template and could be made a bit DRYer using foreign input sets (like in package actions in katello[1])

[1] - https://github.com/Katello/katello/blob/master/app/views/foreman/job_templates/remove_package.erb

Reply:
good idea. will have a look on it. thanks.